README.md

```diff
@@ -35,7 +35,8 @@ You will need an authorization token to create a channel on Kolibri Studio. In o
 
 ### Step 2: Initializing the Channel ###
 
-To run the Ricecooker, you must include a `construct_channel` method in your sushi chef file that returns a ChannelNode object. This function will be responsible for building the structure of your channel.
+To run the Ricecooker, you must include `create_channel` and `construct_channel` methods in your sushi chef file.
```

Comment:
Why do we need both `create_channel` and `construct_channel`? Couldn't the `construct_channel` do both tasks as before?